x/stake/keeper.go

```diff
@@ -777,8 +791,13 @@ func (k Keeper) Delegation(ctx sdk.Context, addrDel sdk.Address, addrVal sdk.Add
 	return bond
 }
 
+// Returns self as it is both a validatorset and delegationset
+func (k Keeper) GetValidatorSet() sdk.ValidatorSet {
```

Comment:
There are probably cleaner ways to implement this pattern, but this is fine for now.